### PR TITLE
ops(ci-cd, standards): added some github actions and forcing some standards

### PR DESCRIPTION
**Pull request by wgmouton**
Please refer to this #252 as this is merely one to keep some things separate until they are ready.

### CODE REVIEW

**Comment by demondayza** [comment]
- Loving the CI / CD Stuff
- Great work so far on the Golang, Please ensure that the existing comments if any are still valid according to the new changes.

**Comment by ig-shaun** [comment]
Willem, Please consider upgrade to 0.45 as there has been a significant upgrade in the Cosmwasm module into this release.

**Comment by wgmouton** [comment]
> Willem, Please consider upgrade to 0.45 as there has been a significant upgrade in the Cosmwasm module into this release.

@ig-shaun Please see #252 as that is the pr that will contain the upgrade.

File: README.md [hunk 1]
```diff
@@ -6,6 +6,18 @@
...
 
+## Notice
+The IXO team is in the process of upgrading this repository better align with our new standards and goals. Exciting things will soon come, but not without first making some fundamental and overdue changes that might cause confusion for some.
+
+### Mini Changelog
+- Upgraded to cosmos-sdk 0.45
+- Introduction of github actions to help automate some tasks. (Note this will be improved as we get more functionality in place)
+- The `master` branch was renamed to `main` and will no longer represent the latest stable version but rather the next feature release. 
+- As the project is still in active developmet, we thought it best to rename all are releases from version `v1.x.x` to `v0.x.x`. This would make [`v0.17.0`](https://github.com/ixofoundation/ixo-blockchain/releases/v0.17.0) our last stable version.
+- Going forward all releases will follow clear samantic versioning guidelines and all stable releases will have a release branch dedicated to it. For example the release `v0.17.0` will associated with the branch `release/v0.17.x` and all bugfixes related to this release should be made against this branch as well as all upstream branches if deemed relevant.
```
Comment (wgmouton) [inline README.md:17]: @demondayza Please review this part.

**Review by demondayza — COMMENTED** [review]
Remove may cause some confusion.

**Pull request merged**